website/versioned_docs/version-2.1/api/makeRenderRestHook.md

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
these new files are just because they didn't change from 2.0 previously so it didn't create a copy